torchrl/objectives/dqn.py

```diff
@@ -231,10 +231,6 @@ def in_keys(self):
             self._set_in_keys()
         return self._in_keys
 
-    @in_keys.setter
-    def in_keys(self, values):
-        self._in_keys = values
-
```

Comment:
this change you requested makes it bc-breaking

Reply:
do you want me to still allow it but warn?

Reply:
no if tests pass without needing it we're good without it